Wordle.Api/Wordle.Api/Data/Player.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+
+namespace Wordle.Api.Data
+{
+    public class Player
+    {
+        public int PlayerId { get; set; }
+
+        [Required]
+        public string Name { get; set; } = string.Empty; // Initialize Name with an empty string
+
+        public int Score { get; set; }
+        public int GameCount { get; set; }
+
+        public double AverageAttempts { get; set; }
```

Comment:
Unless you're calculating this client-side, you will have problems updating this. I'd recommend you have another var called TotalAttempts, and instead make AverageAttempts return TotalAttempts / GameCount. Maybe Score is TotalAttempts?